src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
- [ ] Your app structure is looking great! However, the goal of this part of the activity is to create the list dynamically, this means using JS to:

- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
i. description [string].
ii. completed [bool].
iii. index: [number].

- Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

Please address this requirements as this is the goal of this activity.

Reply:
Noted thank you! I implemented the same. I had thought it was all about the skeleton of the interface at first.